source/fundamentals/collations.txt

```diff
@@ -0,0 +1,539 @@
+   ``import com.mongodb.client.model`` package for convenience.
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CollationCollectionExample.java
+   :language: java
+   :dedent:
+   :start-after: start findOneAndDelete
+   :end-before: end findOneAndDelete
+
+After you run the operation above, your output should resemble the
+following:
+
+.. code-block:: none
+
+   Deleted document: {"_id": 3, "a": "179 bananas"}
+
+The numeric value of the string "179" is greater than the number 100, so
+the above document is the only match.
+
+If we perform the same operation without the numerical ordering collation
+on the original collection of three documents, the filter matches all of
+our documents since "100" comes before "16", "84", and "179" when ordering
+by binary collation.
+
+For more information on the methods used in this section, see the following
+API documentation pages:
+
+- :java-docs:`findOneAndDelete <apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#findOneAndDelete(org.bson.conversions.Bson)>`
+- :java-docs:`FindOneAndDeleteOptions <apidocs/mongodb-driver-core/com/mongodb/client/model/FindOneAndDeleteOptions.html>`
+- :java-docs:`Filters <apidocs/mongodb-driver-core/com/mongodb/client/model/Filters.html>`
+- :java-docs:`Sorts <apidocs/mongodb-driver-core/com/mongodb/client/model/Sorts.html>`
+
+Aggregation Example
+```````````````````
+
+This section demonstrates how you can specify a collation in an aggregation
+operation. In an aggregation operation, you can specify a series
+aggregation stages which is collectively called the aggregation pipeline. To
+perform an aggregation, call the ``aggregate()`` method on a
+``MongoCollection`` object.
+
+To specify a collation for an aggregation operation, call the ``collation()``
+method on the ``AggregateIterable`` returned by the aggregation operation.
+Make sure to specify a sort aggregation stage on which to apply the
+collation in your aggregation pipeline.
+
+The following example shows how we can construct an aggregation pipeline on
+the :ref:`example collection <collation-example-collection>` and apply
+a collation by specifying the following:
+
+- A group aggregation stage using the ``Aggregates.group()`` helper to
+  identify each document by the ``first_name`` field and use that value as
+  the ``_id`` of the result.
+- An accumulator in the group aggregation stage to sum the number of
+  instances of matching values in the ``first_name`` field.
+- Apply an ascending sort to the ``_id`` field of the output documents of
+  the prior aggregation stage.
+- Construct a collation object with the German phonebook collation.
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CollationCollectionExample.java
+   :language: java
+   :dedent:
+   :start-after: start aggregationExample
+   :end-before: end aggregationExample
+
+The code above outputs the following documents:
+
+.. code-block:: json
+   :copyable: false
+
+   {"_id": "Günter", "nameCount": 1}
```

Comment:
The example would be more compelling if one of the groups had more than one member.  Maybe an example that uses a collation that would collapse both `Gunter`s into a single group?  Is that possible?

Reply:
I'll try to find a different collation that might ignore umlauts.